from .base_exception import BaseException


class NotFoundException(BaseException):
    pass
